src/modules/authentication/electron.oidc.authentication.service.ts

```diff
@@ -0,0 +1,242 @@
+
+    if (userIsNotLoggedIn) {
+      return this._getDummyAccessToken();
+    }
+
+    return this._tokenObject.accessToken;
+  }
+
+  public async getIdentity(): Promise<IIdentity | null> {
+
+    const token: string = this.getAccessToken();
+
+    const request: Request = new Request(`${environment.openIdConnect.authority}/connect/userinfo`, {
+      method: 'GET',
+      mode: 'cors',
+      referrer: 'no-referrer',
+      headers: {
+        'Access-Control-Allow-Origin': '*',
+        'Content-Type': 'application/json',
+        Authorization: `Bearer ${token}`,
+      },
+    });
+
+    const response: Response = await fetch(request);
+
+    if (response.status === UNAUTHORIZED_STATUS_CODE) {
+      return null;
+    }
+
+    return response.json();
+  }
+
+  public async checkUserInfo(): Promise<void> {
+    const token: string = this.getAccessToken();
+
+    const request: Request = new Request(`${environment.openIdConnect.authority}/connect/userinfo`, {
+      method: 'GET',
+      mode: 'cors',
+      referrer: 'no-referrer',
+      headers: {
+        'Access-Control-Allow-Origin': '*',
+        'Content-Type': 'application/json',
+        Authorization: `Bearer ${token}`,
+      },
+    });
+
+    try {
+      const response: Response = await fetch(request);
+
+      if (response.status === UNAUTHORIZED_STATUS_CODE) {
+        this._tokenObject = undefined;
+
+        this._logoutUserFromAllSolutions();
+      }
+
+    } catch (error) {
+      this._notificationService.showNotification(NotificationType.WARNING, 'The identity server may be offline!');
+      this._tokenObject = undefined;
+
+      this._logoutUserFromAllSolutions();
+    }
+
+  }
+
+  private async _isIdentityServerReachable(): Promise<boolean> {
+    const request: Request = new Request(`${environment.openIdConnect.authority}/.well-known/openid-configuration`, {
+      method: 'GET',
+      mode: 'cors',
+      referrer: 'no-referrer',
+      headers: {
+        'Access-Control-Allow-Origin': '*',
+        'Content-Type': 'application/json',
+      },
+    });
+
+    let response: Response;
+
+    try {
+
+     response = await fetch(request);
+    } catch (error) {
+      if (error.message === 'Failed to fetch') {
+        return false;
+      }
+    }
+
+    if (response.status === IDENTITY_SERVER_AVAILABLE_SUCCESS_STATUS_CODE) {
+      return true;
+    }
+
+    return false;
+  }
+
+  // TODO: The dummy token needs to be removed in the future!!
+  // This dummy token serves as a temporary workaround to bypass login. This
+  // enables us to work without depending on a full environment with
+  // IdentityServer.
+  private _getDummyAccessToken(): string {
+    const dummyAccessTokenString: string = 'dummy_token';
+    const base64EncodedString: string = btoa(dummyAccessTokenString);
+    return base64EncodedString;
+  }
+
+  private _persistTokenObject(): void {
+    window.localStorage.setItem('tokenObject', JSON.stringify(this._tokenObject));
```

Comment:
Probably we need to have a namespace like thing:

```suggestion
    window.localStorage.setItem('bpmnstudio.tokenObject', JSON.stringify(this._tokenObject));
```

I'm afraid, we might collide with other applications.